README.md

```diff
@@ -6,39 +6,45 @@ Open Policy Agent (OPA) is an open source, general-purpose policy engine that en
+- Install the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=tsandall.opa) to get started locally with live diagnostics, debugging and formatting. See [Editor and IDE Support](https://www.openpolicyagent.org/docs/edge/editor-and-ide-support/) for other supported editors.
+- Go to the [OPA Documentation](https://www.openpolicyagent.org/docs/latest/) to
+  learn about the Rego language as well as how to deploy and integrate OPA.
+- Check out the learning resources in the [Learning Rego](https://www.openpolicyagent.org/ecosystem/learning-rego/) section of the ecosystem directory.
+- Watch OPA's [YouTube channel](https://www.youtube.com/channel/UClDMRN5HlqD3di5MMf-SV4A) for videos about OPA, or see the list of [presentations](#presentations) below.
+- Read the project updates on the [OPA Blog](https://blog.openpolicyagent.org/)
+  or check out the [OPA Roadmap](https://docs.google.com/presentation/d/16QV6gvLDOV3I0_guPC3_19g6jHkEg3X9xqMYgtoCKrs/edit?usp=sharing) to see a high-level snapshot of OPA features in-progress and planned.
 
-## Want to learn more about OPA?
+## Want to talk about OPA or get support?
 
-- Go to [openpolicyagent.org](https://www.openpolicyagent.org) to get started with documentation and tutorials.
-- Browse [blog.openpolicyagent.org](https://blog.openpolicyagent.org) for news about OPA, community, policy and authorization.
-- Watch OPA's [YouTube](https://www.youtube.com/channel/UClDMRN5HlqD3di5MMf-SV4A) channel for videos about OPA.
-- Try OPA with the [Rego Playground](https://play.openpolicyagent.org) to experiment with policies and share your work.
-- View the [OPA Roadmap](https://docs.google.com/presentation/d/16QV6gvLDOV3I0_guPC3_19g6jHkEg3X9xqMYgtoCKrs/edit?usp=sharing) to see a high-level snapshot of OPA features in-progress and planned.
+- Join the [OPA Slack](https://slack.openpolicyagent.org) to talk to other OPA users and maintainers. See `#help` for support.
+- Check out the [Community Discussions](https://github.com/orgs/open-policy-agent/discussions) to ask questions.
+
+## Interested to learn what others are doing with OPA?
+
+- Browse community projects on the [OPA Ecosystem Directory](http://openpolicyagent.org/ecosystem/) - don't forget to [list your own](https://github.com/open-policy-agent/opa/tree/main/docs#opa-ecosystem)!
 - Check out the [ADOPTERS.md](./ADOPTERS.md) file for a list of production adopters. Does your organization use OPA in production? Support the OPA project by submitting a PR to add your organization to the list with a short description of your OPA use cases!
 
 ## Want to download OPA?
 
-- [Docker Hub](https://hub.docker.com/r/openpolicyagent/opa/tags/) for Docker images.
-- [GitHub releases](https://github.com/open-policy-agent/opa/releases) for binary releases and changelogs.
+- See [Docker Hub](https://hub.docker.com/r/openpolicyagent/opa/tags/) for container images and the [GitHub releases](https://github.com/open-policy-agent/opa/releases) for binaries.
```

Comment:
Maybe link to https://www.openpolicyagent.org/docs/latest/#running-opa?

Reply:
Yeah that makes sense, added in 5f2c7063119b708a6cb7eda849129a6b7823ae0d